acceptance/tests/facts/windows.rb

```diff
@@ -0,0 +1,95 @@
+test_name "Facts should resolve as expected in Windows 2012R2"
+
+#
+# This test is intended to ensure that facts specific to an OS configuration
+# resolve as expected in Windows 2012R2.
+#
+# Facts tested: os, processors, ruby, networking, identity, kernel
+#
+
+confine :to, :platform => "windows-2012"
```

Comment:
@MikaelSmith should have mentioned - I'm running out the door and ran out of time to get other windows versions up - but they're coming (tomorrow) :)